leaf-ui/js/displayAnalysis.js

```diff
@@ -420,15 +450,26 @@ $('#analysis-sidebar').on("click", ".result-elements", function(e){
     // Grab Config and result information
     var configId = e.currentTarget.parentElement.id.split("-")[0];
     var resultIndex = $(e.target).text().split(" ")[1];
-    var currAnalysisConfig = analysisMap.get(configId)
+    var currAnalysisConfig = analysisMap.get(configId);
     var currAnalysisResults = currAnalysisConfig.analysisResults[resultIndex-1];
     analysisRequest = currAnalysisConfig.getAnalysisRequest();
 
     // Update UI accordingly
     $(e.target).css("background-color", "#A9A9A9");
-    $("#"+configId).css("background-color","#A9A9A9")
-    refreshAnalysisUI()
-    displayAnalysis(currAnalysisResults)
+    $("#"+configId).css("background-color","#A9A9A9");
```

Comment:
This is gonna cause a bit of a merge conflict with the naming-feature branch but should be an easy fix on either end